test/algorithms/buffer/buffer_multi_polygon.cpp

```diff
@@ -541,7 +605,7 @@ int test_main(int, char* [])
 #endif
 
 #if defined(BOOST_GEOMETRY_TEST_FAILURES)
-    BoostGeometryWriteExpectedFailures(1, 1, 2, 3);
+    BoostGeometryWriteExpectedFailures(1, 14, 2, 11);
```

Comment:
Because it's related to precision, for float these changes were not necessary.
double/long double add more errors, just because I added the failing cases and part of them are not yet solved.